docs/mv-guide.md

```diff
@@ -207,15 +207,15 @@
       materialized view, if not, the operation is allowed, otherwise operation will be rejected by
       throwing exception.
    3. Partition management command: `ALTER TABLE ADD/DROP PARTITION`. Note that dropping a partition
-      will be allowed only if partition is participating in all indexes associated with fact table.
+      will be allowed only if partition is participating in all materialized views associated with fact table.
```

Comment:
"if partition" can be changed to "if the partition"

Reply:
changed